fastlane_core/lib/fastlane_core/itunes_transporter.rb

```diff
@@ -125,7 +129,7 @@ def parse_line(line, hide_output)
             CredentialsManager::AccountManager.new(user: @user).invalid_credentials
             UI.error("Please run this tool again to apply the new password")
           end
-        elsif $1.include?("Redundant Binary Upload. There already exists a binary upload with build")
+        elsif $1.include?("Redundant Binary Upload. You've already uploaded a build with build")
```

Comment:
`Redundant Binary Upload. You've already uploaded a build with build` -> `Redundant Binary Upload. You've already uploaded a build with this build number (<maybe build number here>)`

Reply:
@taquitos Not sure if you mean the user friendly error message should have the build number? I've added that in now though. If you wanted to check specifically for the build number that is in the IPA we uploaded I'm not sure if Fastlane is unpacking the IPA to get that info at all before uploading it via transporter?

Reply:
Yeah, that message didn't make any sense to me `You've already uploaded a build with build`  would it make more sense as `You've already uploaded a build with the build action` or is it `You've already uploaded a build with a the same build number X`

Reply:
Error message is still broken here @cp1. Maybe you didn't push a commit you made?

Reply:
The actual error message is below, this just seems to match on one of the original error messages here to display the error message below.